src/Transitions/Transition.js

```diff
@@ -0,0 +1,173 @@
+'use strict';
+import React from 'react';
+import TransitionEvents from '../utils/TransitionEvents';
+import classnames from 'classnames';
+
+function omit(obj, keys) {
+  let included = Object.keys(obj).filter( k => keys.indexOf(k) === -1)
+  let newObj = {};
+
+  included.forEach( key => newObj[key] = obj[key] )
+  return newObj;
+}
```

Comment:
Sure but the performance isn't remotely a bottleneck here so it wouldn't have any practical impact on speed here, especially since it's looping through a just few keys, we are talking about differences in nano seconds